idunn/api/geocoder.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional
+
+from fastapi import Query
+from pydantic import BaseModel
+
+from idunn import settings
+from ..geocoder.client import geocoder_client
+
+
+class GeocoderParams(BaseModel):
+    shape: dict = Query({}, title='restrict search inside of a polygon')
+
+def get_autocomplete(
```

Comment:
Please add an empty line before this function (elements must be separated by two empty lines according to PEP).